pilot/pkg/model/envoyfilter.go

```diff
@@ -73,8 +73,12 @@ func convertToEnvoyFilterWrapper(local *Config) *EnvoyFilterWrapper {
 			Match:     cp.Match,
 			Operation: cp.Patch.Operation,
 		}
+		var err error
+		cpw.Value, err = xds.BuildXDSObjectFromStruct(cp.ApplyTo, cp.Patch.Value)
 		// there won't be an error here because validation catches mismatched types
```

Comment:
nit: can you remove/correct this comment?

Reply:
Yeah, this only happens in tests btw